src/ui/layout/search/GrandSearch.vue

```diff
@@ -135,8 +135,11 @@ export default {
             // dropdown is visible, this will collapse the dropdown.
             if (this.$refs.GrandSearch) {
                 const clickedInsideDropdown = this.$refs.GrandSearch.contains(event.target);
-                if (!clickedInsideDropdown && this.$refs.searchResultsDropDown._data.resultsShown) {
-                    this.$refs.searchResultsDropDown._data.resultsShown = false;
+                const clickedPreviewClose = event.target.parentElement && event.target.parentElement.querySelector('.js-preview-window');
+                const searchResultsDropDown = this.$refs.searchResultsDropDown._data;
+                if (!clickedInsideDropdown && searchResultsDropDown.resultsShown && !searchResultsDropDown.previewVisible && !clickedPreviewClose) {
```

Comment:
One thing I notice is that if you click anywhere inside the preview at all (except for the 'X' close button), the search results dropdown will close. I could imagine a scenario where a user needs to look at a preview more closely and thus expands some info, or zooms in on a plot, etc. This would close the search dropdown and the user would have to re-focus the search input.

As a user, I wouldn't expect the search dropdown to close if clicking anywhere inside the preview, as the next action is either going to be dragging the object into a layout, or previewing another object to find the right one. This could become more frustrating the longer the search results are and the more similarly-named the objects are.

In most cases likely a minor inconvenience, but still has a UX impact that should be considered.

Reply:
nice catch!